cloudy_mozdef/cloudformation/mozdef-es.yml

```diff
@@ -17,6 +17,34 @@ Parameters:
     Type: Number
     Description: The number of ElasticSearch nodes in the cluster
 Resources:
+  MozDefElasticSearchSecurityGroup:
```

Comment:
It's not clear because the location of this stanza changed but this adds support for port 80